test/stamp/stamp_test.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
-if [ "`$DATA`" = "12345" ]; then
+git_revision=$(git rev-parse HEAD)
```

Comment:
this test should not execute git commands; this would fail if it were sandboxed, for example.

Reply:
aww, have updated now :)